.github/workflows/main.yml

```diff
@@ -17,20 +17,27 @@ on:
 
 jobs:
   tests:
-    name: ${{ matrix.os }} / ${{ matrix.python-version }}
+    name: ${{ matrix.os }} / ${{ matrix.python-version }} ${{ matrix.suffix }}
```

Comment:
I wonder if this will add a space to the job name/break the required checks.

Reply:
It looks like this doesn't cause any issues. Cool!

Reply:
i wondered that as well, but the jobs still show the `Required` annotation, and highlighting the space between e.g. `3.6 (pull_request)` shows only a single space which is the same as before this PR. maybe github actions trims trailing spaces?